src/proxy_app/router_integration.py

```
@@ -71,6 +71,40 @@ def _initialize_adapters(self):
+            )
+            if _db_file.exists():
+                with open(_db_file, "r") as _f:
+                    _db = _yaml.safe_load(_f) or {}
+                for _p in _db.get("providers", []) or []:
+                    if not isinstance(_p, dict):
+                        continue
+                    _pid = _p.get("id")
+                    _adw = _p.get("active_days_window")
+                    if _pid and isinstance(_adw, dict):
+                        active_days_windows[_pid] = _adw
+        except Exception as _e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)